### DISCUSSION TITLE
Browser stealth mode

### DISCUSSION THREAD

**Discussion by gregpr07** [browser-use/browser-use#81] [Ideas]
Who wants this?

**Comment by paolodina** [comment]
sounds appealing. broadly speaking, how would this work?

**Comment by marcopod** [comment]
was this implemented?

**Comment by zahariash** [comment]
It's a must on many websites.

**Comment by UsamaEjaz0** [comment]
Was this done?

**Reply by pirate** [reply]
yes there is now a `stealth=True` option on `main` and out in `0.2.6`.

`0.2.5` also supports stealth but it's more complicated, you have to pass in a playwright object manually `playwright=(await async_patchright().start())`
